# --- T2-COPYRIGHT-NOTE-BEGIN ---
# This copyright note is auto-generated by ./scripts/Create-CopyPatch.
# 
# T2 SDE: package/.../dietlibc/patches/pkg_man.patch
# Copyright (C) 2006 The T2 SDE Project
# 
# More information can be found in the files COPYING and README.
# 
# This patch file is dual-licensed. It is available under the license the
# patched project is licensed under, as long as it is an OpenSource license
# as defined at http://www.opensource.org/ (e.g. BSD, X11) or under the terms
# of the GNU General Public License as published by the Free Software
# Foundation; either version 2 of the License, or (at your option) any later
# version.
# --- T2-COPYRIGHT-NOTE-END ---

--- ./src/man.c.orig	2006-06-19 00:21:55.000000000 +0200
+++ ./src/man.c	2006-06-19 00:24:43.000000000 +0200
@@ -30,6 +30,9 @@
 #include <errno.h>
 #include <unistd.h>
 #include <locale.h>
+#include <termio.h>
+#define index strchr
+#define rindex strrchr
 
 #ifndef R_OK
 #define R_OK 4
--- ./src/man-config.c.orig	2006-06-19 00:26:33.000000000 +0200
+++ ./src/man-config.c	2006-06-19 00:27:06.000000000 +0200
@@ -32,6 +32,7 @@
 #include "paths.h"
 #include "gripes.h"
 #include "util.h"
+#define rindex strrchr
 
 #define BUFSIZE 4096
 
--- ./src/to_cat.c.orig	2006-06-19 00:29:10.000000000 +0200
+++ ./src/to_cat.c	2006-06-19 00:29:22.000000000 +0200
@@ -1,6 +1,7 @@
 #include <stdio.h>
 #include <stdlib.h>
 #include <string.h>
+#define rindex strrchr
 
 extern char *rindex (const char *, int);	/* not always in <string.h> */
 
--- ./src/manpath.c.orig	2006-06-19 00:29:57.000000000 +0200
+++ ./src/manpath.c	2006-06-19 00:30:16.000000000 +0200
@@ -26,6 +26,8 @@
 #include <unistd.h>
 #include <sys/types.h>
 #include <sys/stat.h>
+#define index strchr
+#define rindex strrchr
 
 /* not always in <string.h> */
 extern char *index(const char *, int);
